sections/2-code.tex

```diff
@@ -96,6 +129,8 @@ \subsubsection{Current Ecosystem}
 \end{itemize}
 
 To demonstrate how several of the affiliated packages can be used together with \sunpypkg in a scientific workflow, we show an example in \autoref{fig:affiliated-package-showcase} of how coronal loop structures can be analyzed using potential magnetic field extrapolations and multi-point extreme ultraviolet (EUV) observations.
+\added[id=WTB]{We have included a Jupyter notebook that illustrates each step of this workflow in the \github repository that accompanies this paper.}\footnote{The \github repository for this paper, including the complete text and all code to generate Figure 2, can be found at \url{https://github.com/sunpy/sunpy-frontiers-paper}}
```

Comment:
```suggestion
\added[id=WTB]{We have included a Jupyter notebook that illustrates each step of this workflow in the \github repository that accompanies this paper.}\footnote{The \github repository for this paper, including the complete text and all code to generate Figure \autoref{fig:affiliated-package-showcase}, can be found at \url{https://github.com/sunpy/sunpy-frontiers-paper}}
```
I wonder if this will break the build.